modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/controller/TaskController.scala

```diff
@@ -127,8 +127,14 @@ class TaskController {
           case Some(smi: SpreadsheetMediaInformation) =>
             val SpreadsheetMediaInformation(idField, inputFields, outputFields, pointFields, decimals) = smi
             val config = this.checkerConfigurationService.getAll(cid, tid).head
-            val path = this.storageService.pathToMainFile(config.id).get.toString
-            val spreadsheetFile = new File(path)
+            val tempFile: String = "tmp"
+            var spreadsheetFile: File = new File(tempFile)
+            if (config.isInBlockStorage) {
+              storageService.getFileFromBucket("tasks", s"${task.id}/main-file", tempFile)
```

Comment:
I would store the name of the `main-file` in as a static member of a global object